src/custom/components/Settings/SettingsMod.tsx

```diff
@@ -129,7 +130,19 @@ export default function SettingsTab({ className, placeholderSlippage, SettingsBu
 
   const [expertMode, toggleExpertMode] = useExpertModeManager()
   //mod
-  const [recipientToggleVisible, toggleRecipientVisibility] = useRecipientToggleManager()
+  const [recipientToggleVisible, toggleRecipientVisibilityAux] = useRecipientToggleManager()
+  const toggleRecipientVisibility = useCallback(
+    (value?: boolean) => {
+      const newRecipientToggleVisibilityValue = value ?? !recipientToggleVisible
```

Comment:
Is this our constant name? If it is can we change it to something easier to understand?

Reply:
I thought this came from Uni, but we added those names

@alfetopito do you know why we have mods in `src`? 
<img width="1324" alt="image" src="https://user-images.githubusercontent.com/2352112/171867616-ed0ff938-d384-430b-aac8-89ae0c98daa8.png">

Reply:
I see it was added by Maria, I think someone in Protofire should amend it

Reply:
David, i just simplified some local names, although this shouldn't be part of this PR

Reply:
Yeah, likely to have slipped through my reviews when Protofire was getting started with CowSwap modding

Reply:
Created an issue for it in Protofire's bucket https://github.com/cowprotocol/cowswap/issues/636